lib/spaceship/tunes/tunes_client.rb

```diff
@@ -585,29 +585,29 @@ def submit_testflight_build_for_review!(app_id: nil, train: nil, build_number: n
                                             review_password: nil,
                                             encryption: false)
 
-      build_info = get_build_info_for_review(app_id: app_id, train: train, build_number: build_number)
+      build_info = get_build_info_for_review(app_id: app_id, train: train, build_number: build_number, platform: platform)
       # Now fill in the values provided by the user
 
       # First the localised values:
       build_info['testInfo']['details'].each do |current|
-        current['whatsNew']['value'] = changelog
-        current['description']['value'] = description
-        current['feedbackEmail']['value'] = feedback_email
-        current['marketingUrl']['value'] = marketing_url
-        current['privacyPolicyUrl']['value'] = privacy_policy_url
+        current['whatsNew']['value'] = changelog if changelog
+        current['description']['value'] = description if description
+        current['feedbackEmail']['value'] = feedback_email if feedback_email
+        current['marketingUrl']['value'] = marketing_url if marketing_url
+        current['privacyPolicyUrl']['value'] = privacy_policy_url if privacy_policy_url
```

Comment:
All these are marked optional now, because iTC will automatically copy the values from the previous TestFlight build. So for new builds in the same build_train, it is not necessary to provide these fields over and over.